__tests__/app.test.js

```diff
@@ -1,9 +1,22 @@
 const endpointsJson = require("../endpoints.json");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const app = require("../app");
+
 /* Set up your test imports here */
 
 /* Set up your beforeEach & afterAll functions here */
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
 
-describe("GET /api", () => {
+describe.skip("GET /api", () => {
```

Comment:
Please do not submit any tests with any .skip's/.only's as we dont know if this actually works, and you should only be pushing working code up to github